editor/import/scene_import_settings.cpp

```diff
@@ -530,6 +535,12 @@ void SceneImportSettings::_load_default_subresource_settings(HashMap<StringName,
 	}
 }
 
+void SceneImportSettings::set_generate_collider(ResourceImporterScene::InternalImportCategory &category) {
+	if (category == ResourceImporterScene::INTERNAL_IMPORT_CATEGORY_MESH_3D_NODE) {
+		generate_collider = true;
+	}
```

Comment:
I would move the if statement out of this function and rename the function ``request_generate_collider``

So it would be called like:
```
if (category == ResourceImporterScene::INTERNAL_IMPORT_CATEGORY_MESH_3D_NODE) {
    SceneImportSettings::get_singleton()->request_generate_collider();
}
```

To me that is more clear about what the function is doing, and it matches the pattern already used in ``_set()``

Reply:
Fair enough, let me make the change and resubmit